demo/package.json

```diff
@@ -7,7 +7,8 @@
     "start": "snowpack dev",
     "build": "snowpack build",
     "lode-cli": "node ../cli/index.js",
-    "serve": "serve ./build -p 8080"
+    "serve": "serve ./build -p 8080",
+    "setup": "yarn install && yarn lode-cli && yarn upgrade lode-three"
```

Comment:
IMO this setup should also call setup in the cli?

Reply:
Even though that would lead to multiple calls when run from the parent directory?

Reply:
It's fine for now as we don't have a nice monorepo setup, but this is kind of unintuitive.
There would be no drawback when calling the setup of the cli 😄